src/main/java/com/gd/clinic/security/entity/User.java

```diff
@@ -0,0 +1,61 @@
+package com.gd.clinic.security.entity;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Type;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+@Component
+@Entity
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Table(name = "Users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
have you tested this? as far as I remember it doesn't work so easy, you need to add something like this
```
@GeneratedValue(generator = "UUID")
@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
@Type(type = "pg-uuid")
```

jfyi, if it works now, leave as it is